### DISCUSSION TITLE
bison: m4 subprocess failed

### DISCUSSION THREAD

**Discussion by fswarbrick** [zopencommunity/meta#826] [Porting]
Trying to port gpm (General Purpose Mouse package), https://github.com/telmich/gpm, and I'm getting the following during the make step:
```
config.status: executing libtool commands
touch src/.depend # to prevent unecessary warnings
make -C src dep
make[1]: Entering directory '/u/dvfjs/repos/gpm/src'
bison -y  /u/dvfjs/repos/gpm/src/prog/gpm-root.y && mv y.tab.c prog/gpm-root.c
bison: m4 subprocess failed
make[1]: *** [Makefile:149: prog/gpm-root.c] Error 1
make[1]: Leaving directory '/u/dvfjs/repos/gpm/src'
make: *** [Makefile:33: dep] Error 2
```
Any thoughts?

**Comment by IgorTodorovskiIBM** [comment]
I believe bison accepts `--debug` as an option for more information. 
I vaguely recall seeing this issue, which I believe was fixed via this PR https://github.com/ZOSOpenTools/bisonport/pull/13 and using the z/OS Open Tools m4. Are you using /bin/m4 or zot's m4?

**Reply by fswarbrick** [reply]
I'm using z/OS Tools version.  There is not, in fact, a version of bison or m4 provided with z/OS (there is yacc).  Anyway, the --debug option only traces the grammar, and it's not even getting that far.
Both bison and m4 look to be the most recent z/OS Tools version.
